cmd/build.go

```diff
@@ -18,11 +18,13 @@ import (
 	"bytes"
 	"fmt"
 	"os"
+	"path"
 	"strings"
 	"text/template"
 	"time"
 
 	shell "github.com/progrium/go-shell"
+	"github.com/prometheus/promu/util/sh"
```

Comment:
nits: we generally like to split internal and external packages. Could you move the `ulil/sh` at the end of the import section with a empty line between external and internal packages.